gnucash/report/reports/example/sample-report.scm

```diff
@@ -0,0 +1,485 @@
+               (gnc:html-table-append-row! table (list (symbol->string cell))))
+             list-val)
+            (gnc:html-table-set-style! table "table"
+             'attribute (list "style" "width:200px"))
+            (gnc:html-table-set-caption! table 
+                                         (G_ "List items selected"))
+            (gnc:html-document-add-object! document table))
+          (let ((txt (gnc:make-html-text)))
+            (gnc:html-text-append!
+             txt
+             (gnc:html-markup-p (G_ "(You selected no list items.)")))
+            (gnc:html-document-add-object! document txt)))
+      
+      ;; here's a bullet list of accounts.  We can mark up the
+      ;; account name with an <a></a> anchor with a special HREF to
+      ;; open a Gnucash register when the link is clicked.  What you
+      ;; need to do is pass the HREF "gnc-register:account=My
+      ;; Account Name" to html-markup-anchor.  The account name
+      ;; passed must be the "full" account name that you get from
+      ;; gnc-account-get-full-name.  You should build this url using
+      ;; (gnc-build-url ...)
+      ;;
+      ;; html-markup-anchor takes the link to jump to as its first
+      ;; arg and then puts the remaining args in the body of the
+      ;; link).
+      ;;
+      ;; html-markup-ul makes a "<ul>" unnumbered list, and takes as
+      ;; its one argument a list of items to put in <li> blocks.
+      (if (not (null? accounts))
+          (gnc:html-document-add-object!
+           document
+           (gnc:make-html-text
+            (gnc:html-markup-ul
+             (map 
+              (lambda (acct)
+                (gnc:html-markup-anchor 
+                 (gnc-build-url URL-TYPE-REGISTER
+                   (string-append "account=" 
+                     (gnc-account-get-full-name
+                        acct))
+                     "")
+                 (xaccAccountGetName acct)))
+              accounts))))
+          (gnc:html-document-add-object!
+           document
+           (gnc:make-html-text
+            (gnc:html-markup-p (G_ "You have selected no accounts.")))))
+            
+      (gnc:html-document-add-object! 
+       document 
+       (gnc:make-html-text 
+        (gnc:html-markup-anchor (gnc-build-url URL-TYPE-HELP "gnucash-guide" "") (G_ "Display help"))))
+
+      (gnc:html-document-add-object! 
+       document 
+       (gnc:make-html-text 
+        (gnc:html-markup-p (G_ "Have a nice day!"))))
+      
+      document)))
+
+;; ------------------------------------------------------------------
+;; Here we define the actual report with gnc:define-report
+;; ------------------------------------------------------------------
+(gnc:define-report
+ 
+ ;; The version of this report.
+ 'version 1
+ 
+ ;; The name of this report. This will be used, among other things,
+ ;; for making its menu item in the main menu. You need to use the
+ ;; untranslated value here!
+ 'name (N_ "Options Example Report")

```

Comment:
Can you reply to first commit comment edit?
`'name (N_ "Sample Report")`

Reply:
Since you rather banged on about it in the mailing list I'd think you'd want the name and filename to be the same. I prefer having "report options" in there some how even though it's a bit cumbersome. Otherwise it's not that important.

Reply:
Done as Options Report

Reply:
Following commit set name and menu-name to Sample Report